spec/models/reports/facility_state_spec.rb

```diff
@@ -5,16 +5,8 @@
     it { should belong_to(:facility) }
   end
 
-  # TODO: extract this into a common utility
   around do |example|
-    # We need to enforce a known time for this test, otherwise we will have intermittent failures. For example,
-    # if we use live system time, many of these specs will fail after 18:30 UTC (ie 14:30 ET) when on the last day of a month,
-    # because that falls into the next day in IST (our reporting time zone). So to prevent confusing failures for
-    # developers or CI during North American afternoons, we freeze to a time that will be the end of the month for
-    # UTC, ET, and IST. Timezones! 🤯
-    Timecop.freeze("June 30 2021 23:00 IST") do
-      example.run
-    end
+    freeze_time(example)
```

Comment:
can we think of a better name?

Reply:
I've used `freeze_time_for_reporting_specs` for now, but still not convinced that's good.